src/main/java/core/basesyntax/Main.java

```diff
@@ -0,0 +1,12 @@
+package core.basesyntax;
+
+import core.basesyntax.controller.ConsoleHandler;
+
+public class Main {
+    public static void main(String[] args) {
+        ConsoleHandler handler = new ConsoleHandler();
+        System.out.println("Print 'value, risk' for your bet."
+                + "For quit type 'quit'.");
```

Comment:
In my humble opinion I would place the `message` inside the handler - so it would show up only when the  `handle()` is invoked.

Reply:
okay, moved to handler class